youtube_dl/extractor/zingmp3.py

```diff
@@ -46,36 +46,46 @@ def _extract_item(self, item, page_type, fatal=True):
                 })
             formats.append(f)
 
-        cover = item.get('cover')
+        artist = None
```

Comment:
Consider using the safe extraction method `try_get` from `utils.py`.
```suggestion
        artist = try_get(item, lambda x: x['artist']['name'], compat_str)
```